docs/cdp/cdp_activation/cdp_activation.md

```diff
@@ -0,0 +1,16 @@
+---
+description: Step-by-step activation procedure of Ibexa CDP.
+page_type: landing_page
+---
+
+# Adctivation
+
+Follow step-by-step activation procedure of [[= product_name_cdp =]].
+
+Activation includes configuration, data export and adding Client-side Tracking.
+
```

Comment:
Also, I think we need a bit of an explanation here. Below you mention Data Manager, dashboard, etc. Where are those places available? What does one need to access these places? Activation procedure requires getting information from these places, and for me it is unclear how I can obtain access to these places. I assume one has to get the address and have an account set up by our support, but it is a lucky guess. I'd add a sentence or two of prerequisites, something along the lines of:

```suggestion

!!! note

    Before you can proceed with activation, make sure you have [obtained the credentials to CDP dashboard](cdp_installation.md#register-in-ibexa-cdp-dashboard) and became acquainted with the account details published there."

```

Reply:
Not a good place for 'note'.